sys-fabric/libibumad/libibumad-1.3.10.2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2019 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+DESCRIPTION="OpenIB User MAD lib functions which sit on top of the user MAD kernel modules"
+HOMEPAGE="https://www.openfabrics.org"
+LICENSE="|| ( BSD GPL-2 )"
+SRC_URI="https://www.openfabrics.org/downloads/management/${P}.tar.gz"
+
+SLOT="0"
+
+KEYWORDS="~amd64 ~x86 ~amd64-linux"
+IUSE="+pic static"
+
+#DEPEND="${RDEPEND}"
```

Comment:
Let's not add commented code in ebuilds unless it serves some purpose, I don't think commenting this here adds any value.